docs/documents/DevOps/workflows.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Workflows
+nav_order: 20
+parent: DevOps
+layout: page
+---
+
+# Workflows
+
+## Overview
+Workflows are orchestraited via [github actions](https://docs.github.com/en/actions) which build, test and deliver our code to the [GOV.UK PaaS](https://www.cloud.service.gov.uk/).
+
+Guidance about the use of Github Actions can be found on the [DfE Technical Guidance](https://technical-guidance.education.gov.uk/infrastructure/automation/github-actions/#installation) page.
```

Comment:
Link to top of the page: https://technical-guidance.education.gov.uk/infrastructure/automation/github-actions/